runner/unitTestRunner.go

```diff
@@ -0,0 +1,98 @@
+package runner
+
+import (
+	"encoding/json"
+	"encoding/xml"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"path"
+
+	"github.com/coduno/app/models"
+	"github.com/coduno/piper/docker"
+	"github.com/coduno/piper/util"
+)
+
+// UnitTestResult holds the unit test result created by junit
+type UnitTestResult struct {
+	Tests    string   `xml:"tests,attr"`
+	Failures string   `xml:"failures,attr"`
+	Errors   string   `xml:"errors,attr"`
+	TestCase TestCase `xml:"testcase"`
+}
+
+// TestCase holds a test case created by junit
+type TestCase struct {
+	Name    string  `xml:"name,attr"`
+	Time    string  `xml:"time,attr"`
+	Failure Failure `xml:"failure"`
+}
+
+// Failure holds a failure created by junit
+type Failure struct {
+	Message string `xml:"message,attr"`
+}
+
+const testResultsFileName = "/build/test-results/TEST-com.coduno.TestApplication.xml"
+
+// RunUnitTest is the struct to be passed to the GeneralRun function
+type RunUnitTest struct {
```

Comment:
if I pass `RunUnitTest.computeResults` direclty let`s not mind the error that i`ll get when compiling but in the GeneralRun function it will call `RunUnitTest.computeResults.computeResults`